src/components/ProfileList/ProfileData.ts

```diff
@@ -2,72 +2,141 @@ export interface ProfileListItems {
   id: number
   name: string
   seniority: string
-  location: string
-  technology: string
-  technology2: string
-  technology3: string
+  stack: string
   others: string
+  country: string
+  city: string
+  remote: string
+  technology: string[]
+  employmentType: string
+}
+
```

Comment:
**nitpick:** It's not a component so the file should be named with kebab case: `profile-data.ts`